config/creatures.cfg

```diff
@@ -13,9 +13,6 @@
 #                     workers will dig and claim tiles. When idle, fighters with a digRate > 0 can dig tiles as indicated
```

Comment:
> IMHO, the best is to have everything at the same size.

To be completely honest, I do also think it's even simpler and may fix bugs by design. :) 
As long as it is planned to have a wiki page about this, I'm fine. Mind creating an issue about this?